### DISCUSSION TITLE
How to  add  onkeypress="return check(event,value)" in input ???

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6372] [Help]
![Edit_phone_Number_only_0](https://user-images.githubusercontent.com/29665710/164260281-6916df7f-611e-4341-8077-ab45bc6eced3.png)
![Edit_phone_Number_only_1](https://user-images.githubusercontent.com/29665710/164260311-97b490c5-1612-4377-b6be-eb7efcc6f6e0.png)

**Comment by stixoffire** [comment]
Trying to determine what you really want to do an onkey press event there for ?
Because you can add a validator to the field to make sure they are numbers or even if they are numbers you would allow etc.. and you can also add a phone number formatter 

Search for Field Validator or Formatter (Northwind project has an example of the phone formatter)  ..
